geps/gep-1713/index.md

```diff
@@ -969,3 +969,585 @@ Prior Discussions:
+    - The parent gateway has the following status :
+      ```
+      conditions:
+      - type: Accepted
+        status: True
+        reason: Accepted
+      - type: Programmed
+        status: True
+        reason: Programmed
+      listeners:
+      - name: gw-8080
+        conditions:
+        - type: Accepted
+          status: True
+          reason: Accepted
+        - type: Programmed
+          status: True
+          reason: Programmed
+        - type: Conflicted
+          status: False
+          reason: NoConflicts
+
+      AttachedListenerSets: 1
+      ```
+
+    - The listener set has the following status :
+      ```
+      conditions:
+      - type: Accepted
+        status: True
+        reason: Accepted
+      - type: Programmed
+        status: True
+        reason: Programmed
+      listeners:
+      - name: ls-8080
+        conditions:
+        - type: Accepted
+          status: False
+          reason: PortUnavailable
+        - type: Programmed
+          status: False
+          reason: PortUnavailable
+        - type: Conflicted
+          status: True
+          reason: HostnameConflict
+      // Other accepted listeners if applicable
+      ```
+
+
+1. A listener `ls-conflicted-8080` on a ListenerSet `ls-conflicted` has a hostname conflict with a listener `ls-accepted-8080` on another ListenerSet `ls-accepted`
+    - The parent gateway has the following status :
+      ```
+      status:
+        AttachedListenerSets: 2
+      ```
+
+    - The `ls-accepted` ListenerSet has the following status :
+      ```
+      conditions:
+      - type: Accepted
+        status: True
+        reason: Accepted
+      - type: Programmed
+        status: True
+        reason: Programmed
+      listeners:
+      - name: ls-accepted-8080
+        conditions:
+        - type: Accepted
+          status: True
+          reason: Accepted
+        - type: Programmed
+          status: True
+          reason: Programmed
+        - type: Conflicted
+          status: False
+          reason: NoConflicts
+      ```
+
+    - The `ls-conflicted` ListenerSet has the following status :
+      ```
+      conditions:
+      - type: Accepted
+        status: True
+        reason: Accepted
+      - type: Programmed
+        status: True
+        reason: Programmed
+      listeners:
+      - name: ls-conflicted-8080
+        conditions:
+        - type: Accepted
+          status: False
+          reason: PortUnavailable
+        - type: Programmed
+          status: False
+          reason: PortUnavailable
+        - type: Conflicted
+          status: True
+          reason: HostnameConflict
+      // Other accepted listeners if applicable
+      ```
+
+1. A listener on a ListenerSet without a defined port
+    - If the implementation supports `ListenerSetDynamicPortAssignment` :
+      - The listener must have the following status :
+        ```
+        name: listener-without-port
+        port: <unique dynamically assigned port value>
```

Comment:
Doesn't need to be resolved now, but we will need to come back to what constitutes a "unique" port. If the Protocol is `HTTPS`, does this port need to be unique if the `hostname` is distinct? (That would _substantially_ complicate implementing this, and may be a bad idea. But if we are ruling it out, we need to be explicit about that).